compiler/test/runnable/test23387.d

```diff
@@ -0,0 +1,29 @@
+/* EXTRA_FILES: imports/maker.i imports/freer.i
+ */
+
+// https://issues.dlang.org/show_bug.cgi?id=23387
+
+/+ maker.i
+typedef struct Foo *FooRef;
+struct Foo {
+    int x;
+};
+FooRef make_foo(void);
++/
+import imports/maker;
```

Comment:
`import imports_._maker;`

Reply:
what does that do?

Reply:
you have a slash instead of a dot

Reply:
That's what I get for switching back and forth between C and D!